mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -111,6 +112,7 @@ public FakeMailContext build() {
             private MimeMessage msg;
             private Map<String, Serializable> attributes = new HashMap<String, Serializable>();
             private Optional<String> state = Optional.absent();
+            private boolean fromMailet = false;
```

Comment:
I don't get your logic. EIther it's false or true, I don't see the value of adding a Optional here